apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -41,6 +42,102 @@ const TeacherNavigationBar: React.FunctionComponent = () => {
     }
   }, [sections, selectedSectionId]);
 
+  const courseContent = (
+    <>
+      <Typography
+        semanticTag={'h2'}
+        visualAppearance={'overline-two'}
+        className={styles.sectionHeader}
+      >
+        {i18n.courseContent()}
+      </Typography>
```

Comment:
```suggestion
      getSectionHeader{i18n.courseContent())
```
optional nit: deduplicate

Reply:
Yeah, that is a good move - anything to make this a little less long.